examples/examples_nodejs_test.go

```diff
@@ -227,6 +227,24 @@ func TestAccMNG_withAwsAuth(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccMNG_DiskSize(t *testing.T) {
```

Comment:
These tests need to be added to the test matrix here:
- https://github.com/pulumi/pulumi-eks/blob/master/.github/workflows/release.yml#L451
- https://github.com/pulumi/pulumi-eks/blob/master/.github/workflows/run-acceptance-tests.yml#L429

Just noticed that this is not in the contribution guide (https://github.com/pulumi/pulumi-eks/blob/master/CONTRIBUTING.md). I'll add it there!